test/jit/test_backends.py

```diff
@@ -465,3 +466,121 @@ def test_save_load(self):
     @skipIfRocm
     def test_errors(self):
         self.selective_lowering_test.test_errors()
+
+"""
+Unit Tests for backend with compiler
+This test case and the existing TestBackends are separate because they cover different aspects.
+The actual backend implementation in this test is different.
+It has a simple demo compiler to test the end-to-end flow.
```

Comment:
```suggestion
It has a simple demo compiler to test the end-to-end flow in mobile.
```

Reply:
Could you also apply clang-format on this PR? You could use `clang-format -i test/jit/test_backends.py` from devserver.